src/Database/Query.php

```diff
@@ -1417,6 +1418,10 @@ public function values($data)
      */
     public function update($table)
     {
+        if (!is_string($table)) {
+            throw new InvalidArgumentException('Table must be of type string');
```

Comment:
If we do this, wouldnt appending

    sprtinf(..., got "%s"), gettype($table)

be even more helpful?